parquet-thrift/src/main/java/org/apache/parquet/thrift/ThriftSchemaConverter.java

```diff
@@ -225,14 +225,18 @@ private static ThriftField toThriftField(String name, Field field, ThriftField.R
         final Field listElemField = field.getListElemField();
         type = new ThriftType.ListType(toThriftField(listElemField.getName(), listElemField, requirement));
         break;
+      case UUID:
       case ENUM:
-        Collection<TEnum> enumValues = field.getEnumValues();
-        List<EnumValue> values = new ArrayList<ThriftType.EnumValue>();
-        for (TEnum tEnum : enumValues) {
-          values.add(new EnumValue(tEnum.getValue(), tEnum.toString()));
+        if (field.isEnum()) {
```

Comment:
It looks like there are still enums with field-id 16. It can be that they are using an old Java SDK. This was a reliable way do differentiate between them.